src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -93,6 +93,21 @@ public User getUser(UUID id) {
     return null;
   }
 
+  /** Find and return User list. */
+  public List<User> getUsers() {
+    return users;
+  }
+
+  /** Find and return the User with the given UUID. */
+  public User getUserWithId(UUID id) {
```

Comment:
Same comment about `getMessageWithId`

Reply:
Yes, it is used in `ActivityFeedServlet` in order to find the author of a message and owner of a conversation to include these details in the `output` string of an `Activity`.